readme.md

```diff
@@ -98,6 +98,24 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
 ### Windows:
 - Visual Studio 2022 (Enterprise / Professional / [Community (Free)](https://www.visualstudio.com/vs/community/))
   - Desktop development with C++
+- MSYS2 MinGW Toolchain. The toolchains with supported dependencies are `mingw-x86_64`, `mingw-xi686`, `ucrt-x86_64`, `clang-x86_64`, and `clang-xi686`, each of these require the `$MINGW_PACKAGE_PREFIX` and `msys` and `clangarm-64` are lacking packages for some dependencies
+  - sdl2 (only for UI client)
+  - freetype (can be disabled)
+  - fontconfig (can be disabled)
+  - libzip (>= 1.0)
+  - libpng (>= 1.2)
+  - speexdsp (only for UI client)
+  - curl (only if building with http support)
+  - nlohmann-json (>= 3.6.0)
+  - openssl (>= 1.0; only if building with multiplayer support)
+  - icu (>= 59.0)
+  - zlib
+  - gl (commonly provided by Mesa or GPU vendors; only for UI client, can be disabled)
+  - cmake
+  - benchmark (optional)
+  - innoextract (optional runtime dependency; used for GOG installer extraction during setup)
```

Comment:
I’m missing the libraries for OGG Vorbis and FLAC.

Reply:
Oh, that's my bad, I found the Vorbis and FLAC libraries used, I'm also am gonna assume that the OGG library is `libogg`, the Linux section on the README and the [dependencies table on the wiki](https://github.com/OpenRCT2/OpenRCT2/wiki/Building-OpenRCT2-on-Linux#required-packages-general) is actually missing those libraries too
```suggestion
  - innoextract (optional runtime dependency; used for GOG installer extraction during setup)
  - libogg
  - libvorbis
  - flac
```